dsgrid/config/association_tables.py

```diff
@@ -29,6 +29,10 @@ class AssociationTableRecordModel(DSGBaseModel):
         title="to_id",
         description="Destination mapping",
     )
+    fraction: Optional[float] = Field(
+        title="fraction",
+        description="Fraction of from_id to map to to_id",
+    )
```

Comment:
I don't think users should have to fill a column out if the fraction is always 1.